README.md

```diff
@@ -1,279 +1,146 @@
-```
-
-To see a pretty visual of the graph itself, we can use the dot command from Graphiz. Exit the shell by typing `exit` and then run the following command:
-
-`dot -Tpdf fam.dot -o fam.pdf`
-
-A new file called `fam.pdf` will now appear in the `hello_jac` directory. Open this file to see your graph!
-
-## References
 
-- Official Documentation: https://docs.jaseci.org/
-- Jaseci Bible: https://github.com/Jaseci-Labs/jaseci_bible
+## Getting Started
+
+- [Introducing Jaseci](support/guide/getting_started/introduction.md)
+- [Installing Jaseci](support/guide/getting_started/installation.md)
+    - [Installing on Windows](support/guide/getting_started/installation.md#installing-on-windows)
+    - [Installing on Mac](support/guide/getting_started/installation.md#installing-on-mac)
+    - [Installing on Linux](support/guide/getting_started/installation.md#installing-on-linux)
+    - [Installation for Contributors](support/guide/getting_started/installation.md#installation-for-contributors-of-jaseci)
+- [Jaseci Quickstart](support/guide/getting_started/quickstart.md)
+- [Setting Up Your Editor](support/guide/getting_started/setting_up_your_editor.md)
+- [Writing Your First App](support/guide/getting_started/writing_your_first_app.md)
+- [Understanding JAC Programs](support/guide/getting_started/understanding_jac_programs.md)
+
+
+## The JAC Programming Language Guide
+
+- [Language Overview](support/guide/jac_language_guide/jac_language_overview.md)
+- [Grammar](support/guide/jac_language_guide/jac_grammar.md)
+- [Operators](support/guide/jac_language_guide/operators.md)
+- [Control Flow](support/guide/jac_language_guide/control_flow.md)
+- [Collections](support/guide/jac_language_guide/collections.md)
+- [Nodes](support/guide/jac_language_guide/nodes.md)
+- [Edges](support/guide/jac_language_guide/edges.md)
+- [Graphs](support/guide/jac_language_guide/graphs.md)
+- [Walkers](support/guide/jac_language_guide/walkers.md)
+    - [Traversing a Graph](support/guide/jac_language_guide/traversing_a_graph.md)
+    - [Referencing a Node in Context](support/guide/jac_language_guide/referencing_node_context.md)
+- [Passing Arguments](support/guide/jac_language_guide/passing_arguments.md)
+- [Specifying Operating Context](support/guide/jac_language_guide/specifying_operating_context.md)
+- [Actions](support/guide/jac_language_guide/actions.md)
+    - [Date Actions](support/guide/jac_language_guide/date_actions.md)
+    - [File Actions](support/guide/jac_language_guide/file_actions.md)
+    - [Net Actions](support/guide/jac_language_guide/net_actions.md)
+    - [Rand Actions](support/guide/jac_language_guide/rand_actions.md)
+    - [Request Actions](support/guide/jac_language_guide/request_actions.md)
+    - [Standard Actions](support/guide/jac_language_guide/standard_actions.md)
+    - [Vector Actions](support/guide/jac_language_guide/vector_actions.md)
+    - [Jaseci Actions](support/guide/jac_language_guide/jaseci_actions.md) 
+
+
+## Codelabs and Guides
```

Comment:
There is a more recent canoniCAI codelab at https://github.com/Jaseci-Labs/jaseci/tree/main/examples/CanoniCAI. Can we link that one as well?

Reply:
It is basically similar scope as the chapter 1 - 7 that are currently linked, but re-written to be more hands-on. We can link both and highlight the new one first.